CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to this project since 0.82.0 will be documented in this file.
 
+## [3.3.5] - 2022-05-17
+- Update azure-cli-core
```

Comment:
In general, we will publish a version update for iotedgedev with all the changes from the Hackathon, so continue appending the changes in the Changelog but don't bump the version in the code.